src/components/header/header.tsx

```diff
@@ -1,42 +1,65 @@
 'use client';
+
 import { Box, Link, Tab, Tabs } from '@mui/material';
-import React from 'react';
+import { usePathname } from 'next/navigation';
+import React, { useEffect, useState } from 'react';
 import './header.scss';
 
+enum TabValue {
+  projects = '/projects',
+  joinUs = '/join-us',
+  about = '/about',
+}
+
 function Header() {
+  const [currentPath, setCurrentPath] = useState<string>(TabValue.projects);
+  const pathname = usePathname();
+  useEffect(() => {
+    setCurrentPath(pathname);
```

Comment:
Here you could remove the forward slash.

Reply:
If you can't fine something in the enum value. SetCurrentPath(false), this prevents a weird MUI error.